.github/workflows/linux.yml

```diff
@@ -243,19 +238,6 @@ jobs:
       run: |
         snapcraft upload *.snap --release ${{ needs.BuildAndTest.outputs.channel }}
 
-  # Report result to Bors on `staging` and `trying` branches.
-  Linux:
-    if: ${{ always() && github.event_name == 'push' && contains('refs/heads/staging refs/heads/trying', github.ref) }}
-    needs:
-    - BuildAndTest
-
-    runs-on: ubuntu-latest
-
-    steps:
-    - name: Report failure
-      if: ${{ needs.BuildAndTest.result != 'success' }}
-      run: exit 1
-
```

Comment:
@townsend2010 you'll want to drop the bare "Linux" check from required checks. This was only here so we needn't list all the build types in `bors.toml`.

Reply:
Hey @Saviq, I don't think the bare Linux check is a required check from what I can tell.